tests/Aspire.Hosting.Containers.Tests/ContainerResourceTests.cs

```diff
@@ -86,7 +86,7 @@ public async Task AddContainerWithArgs()
         """
         {
           "type": "container.v0",
-          "image": "none:latest",
+          "image": "none",
```

Comment:
This seems bad

Reply:
If a user explicitly gives us a tag of `latest`, then we should include that in the manifest (i.e. `none:latest`).  If a user doesn't' give us a tag that's more open question.  My take was that the images `none` and `none:latest` are equivalent, so if a user doesn't give us a tag we should treat that by not explicitly including a tag.  Happy to put `latest` back in the manifest if you think that's better.

That said, I haven't put huge amounts of thought into this latest commit - mainly done the obvious change to see what tests break.